bridge/Mixer.h

```diff
@@ -94,12 +94,14 @@ class Mixer
         const std::vector<uint32_t>& audioSsrcs,
         const std::vector<api::SimulcastGroup>& videoSsrcs,
         const std::vector<api::SsrcPair>& videoPinSsrcs,
-        bool useGlobalPort);
+        bool useGlobalPort,
+        bool disableVideo);
```

Comment:
not really necessary as videoSsrcs will be empty vector and then you cannot do video.